config/crd/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+# This kustomization.yaml is not intended to be run by itself,
```

Comment:
For now, all things deployed by the operator, also removed by the operator, who will be responsible to remove webhooks and all certificate CRs?

Reply:
The CR certificates are manged by OLM - not by cert-manager (template for upstream, not used by us)